manifests/init.pp

```diff
@@ -40,16 +40,16 @@
   Stdlib::Absolutepath $config_file,
   String[1] $package_name,
   String[1] $service_name,
-  String[1] $module_prefix,
-  Stdlib::Absolutepath $init_config_file,
-  Hash $init_config_hash,
+  String[0] $module_prefix,
   String[1] $config_file_header,
-  String[1] $package_ensure,
+  Stdlib::Absolutepath $sbin_path,
+  String[1] $package_ensure = 'installed',
```

Comment:
The value in Hiera was the same for all OS.  In such a case it is preferred to put the default value in the manifest so that puppet-strings can integrate the default value in the API documentation.  See caf1aca9ce0854fdfa95d266235e8b11e69a2e27 for the reason of changing `present` to `installed`.

```diff
-  String[1] $package_ensure,
+  String[1] $package_ensure = 'installed',
```